schemas/gdbots/ncr/command/update-labels/latest.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<pbj-schema xmlns="http://gdbots.io/pbj/xsd"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://gdbots.io/pbj/xsd http://gdbots.io/pbj/xsd/schema.xsd">
+
+  <schema id="pbj:gdbots:ncr:command:update-labels:1-0-0">
+    <fields>
+      <field name="add_labels" type="string" rule="list" required="true"/>
```

Comment:
@rsumilang make this `rule="set"` and should we do a pattern on these?  should not be required.